### PR TITLE
Removing flutter dependency

### PR DESCRIPTION
**Pull request by wolfenrain**

### CODE REVIEW

File: .github/workflows/cicd.yml [hunk 1]
```diff
@@ -38,21 +40,16 @@ jobs:
...
       - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: '12.x'
-      - uses: subosito/flutter-action@v1
-      - run: flutter pub get
-      - run: flutter pub run benchmark
+      - uses: cedx/setup-dart@v2
+      - run: pub get
+      - run: pub run benchmark
```
Comment (erickzanardo) [inline .github/workflows/cicd.yml:54]: Shouldn't be dart pub run here as well?

**Review by erickzanardo — APPROVED** [review]
One nit, but LGTM already

File: .github/workflows/cicd.yml (hunk 1 above)
Comment (wolfenrain) [reply .github/workflows/cicd.yml:54]: The `pub` command is exposed on the PATH variable I believe. Wanted to test that, but having some issues with the yaml formatting :-)

Comment (wolfenrain) [reply .github/workflows/cicd.yml:54]: Ah I now got what you mean, in the test job I am running it with `dart pub run test`. I believe I can replace that with `dart run` to have the same effect. I'll update that

Comment (wolfenrain) [reply .github/workflows/cicd.yml:54]: Fixed and working.

**Pull request merged**